src/test/java/domain/lotto/LottoTicketTest.java

```diff
@@ -1,20 +1,19 @@
 package domain.lotto;
 
-import static lotto.view.messages.ErrorMessages.LOTTO_PURCHASE_PRICE_ERROR;
-import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
 
 import lotto.domain.lotto.LottoTicket;
+import lotto.domain.lotto.util.LottoMoney;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 
 public class LottoTicketTest {
 
     @Test
-    @DisplayName("돈이 부족할 경우 로또 티켓 생성시 예외가 발생한다.")
-    public void testLottoTicketMakeWhenLackOfMoney() {
-        assertThatThrownBy(() -> new LottoTicket(-1))
-            .isInstanceOf(IllegalArgumentException.class)
-            .hasMessageContaining(LOTTO_PURCHASE_PRICE_ERROR.getMessage());
+    @DisplayName("금액에 따른 로또 티켓이 생성된다.")
+    void testCreateLottoTicket() {
+        LottoTicket lottoTicket = new LottoTicket(new LottoMoney(100000000));
+        assertThat(lottoTicket.getLottoLineSize()).isEqualTo(100000);
```

Comment:
아래와 같이 변수와 언더바를 사용하면 좀 더 가독성이 좋아질 수 있을 것 같아요ㅎㅎ
```suggestion
        int inputMoney = 100_000_000;
        LottoTicket lottoTicket = new LottoTicket(new LottoMoney(inputMoney));
        assertThat(lottoTicket.getLottoLineSize()).isEqualTo(inputMoney / LottoMoney.LOTTO_LINE_PRICE);
```